ml-conversational-analytic-tool/utils.py

```diff
@@ -0,0 +1,37 @@
+import os
+import ast
+from pathlib import Path
+
+import pandas as pd
+
+EXPORTS_DIR = "exports"
+
+
+def export_to_cvs(export_df: pd.DataFrame, name=None, default_name=None, suffix="_analysis"):
+    """
+    Export DataFrame into csv file with name constructed based on the given name or default_name
+    """
+
+    if not os.path.exists(EXPORTS_DIR):
+        os.mkdir(EXPORTS_DIR)
+
+    if name:
+        _, file_extension = os.path.splitext(name)
+        if not file_extension:
+            name = name + ".csv"
+        file = os.path.join(EXPORTS_DIR, name)
+    elif default_name:
+        file = os.path.join(EXPORTS_DIR, Path(default_name).stem + suffix + ".csv")
+    else:
+        raise Exception("Both `name` and `default_name` are empty. At least one should be given.")
```

Comment:
It's generally not good to raise the base class of Exception. It's better to raise a more specific exception

Reply:
`ValueError` is used instead